zcash_client_sqlite/src/wallet.rs

```diff
@@ -891,48 +891,107 @@ pub(crate) fn update_expired_notes<P>(
     stmts.stmt_update_expired(height)
 }
 
+// A utility function for creation of parameters for use in `insert_sent_output`
+// and `put_sent_output`
+//
+// - If `to` is a Unified address, this is an index into the outputs of the transaction
+//   within the bundle associated with the recipient's output pool.
+// - If `to` is a Sapling address, this is an index into the Sapling outputs of the
+//   transaction.
+// - If `to` is a transparent address, this is an index into the transparent outputs of
+//   the transaction.
+// - If `to` is an internal account, this is an index into the Sapling outputs of the
+//   transaction.
```

Comment:
I don't understand what "this" refers to. The returned tuple? In any case the description doesn't seem to match what the function computes.